pkg/k8shandler/status.go

```diff
@@ -49,3 +54,35 @@ func updateNodeStatus(node *nodeState, dpl *v1alpha1.ElasticsearchStatus) {
 	}
 	dpl.Nodes = append(dpl.Nodes, nodeStatus)
 }
+
+func clusterHealth() string {
+	config := &ExecConfig{
+		namespace:      "openshift-logging",
+		labelName:      "component",
+		labelPrefix:    "elasticsearch",
+		containerName:  "elasticsearch",
+		command:        []string{"es_util", "--query=_cluster/health?pretty=true"},
+		kubeConfigPath: "/home/origin/admin.kubeconfig",
+		masterURL:      "https://ip-172-18-8-136.ec2.internal:8443",
```

Comment:
@ewolinetz are these exposed through the SDK? 

I assume that at least kubeConfig is available through envar, but have no idea about masterURL.

Reply:
Trying to ask around in the `#forum-operators` channel about this. I would imagine its available to it in some way.